CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## v0.1.35 (beta) - Jan 10 2022
```

Comment:
I think this needs to be v0.1.36 since v0.1.35 was released already.

IMHO we should defer to amend the change log until we submit the "release PR" like #389 (which happened to miss the change log update). That way, the release entry does not show up right before we are actually making the release, which ensures that the date is correct.

Reply:
Ah, I see. I was wondering about that. I'll remove it from this PR and write a note to follow up.